"""All custom exceptions used in datalad-next"""

from datalad.runner import CommandError
from datalad.support.exceptions import (
    AccessDeniedError,
    AccessFailedError,
    NoDatasetFound,
    CapturedException,
    IncompleteResultsError,
    DownloadError,
)
